docs/guides/universal-receiver-delegate/set-default-implementation.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_label: 'Set the default Implementation'
+sidebar_position: 1
+---
+
+# Set the default Implementation
+
+:::note
+
+Users deploying their Universal Profiles using the guides that utilize **[lsp-factory](../universal-profile/create-profile.md)** or the **[Browser Extension](../browser-extension/create-a-universal-profile.md)** can skip this guide, as this contract is already deployed and set for their profiles.
```

Comment:
```suggestion
Users deploying their Universal Profiles using the guides that utilise **[lsp-factory](../universal-profile/create-profile.md)** or the **[Browser Extension](../browser-extension/create-a-universal-profile.md)** can skip this guide, as this contract is already deployed and set for their profiles.
```

Reply:
That makes me go into the deep vocab of English 😆 
> In most cases, the correct spelling is utilize. Spelled with an S (utilise) is commonly found in the UK. Both are correct. But utilize is more widely accepted.

Will go with the more widely accepted if you don't mind ;)